### ISSUE TITLE
Unknown failure

### ISSUE THREAD

**Issue by patrickmichalina** [conventional-changelog/standard-version#104]
There is an error that is happening on this project AND semantic-release

It occurs right after the changelog is generated. 

```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Users\\pmichalina\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'ttt',
1 verbose cli   '--',
1 verbose cli   'pre' ]
2 info using npm@3.10.8
3 info using node@v6.3.1
4 verbose run-script [ 'prettt', 'ttt', 'postttt' ]
5 info lifecycle @SecretOrg/kick-ass-project@0.2.8~prettt: @SecretOrg/kick-ass-project@0.2.8
6 silly lifecycle @SecretOrg/kick-ass-project@0.2.8~prettt: no script for prettt, continuing
7 info lifecycle @SecretOrg/kick-ass-project@0.2.8~ttt: @SecretOrg/kick-ass-project@0.2.8
8 verbose lifecycle @SecretOrg/kick-ass-project@0.2.8~ttt: unsafe-perm in lifecycle true
9 verbose lifecycle @SecretOrg/kick-ass-project@0.2.8~ttt: PATH: C:\Users\pmichalina\AppData\Roaming\npm\node_modules\npm\bin\node-gyp-bin;C:\Users\pmichalina\Source\Repos\pjmichalina\kick-ass-project\node_modules\.bin;C:\Users\pmichalina\AppData\Local\GitHub\PortableGit_284a859b0e6deba86edc624fef1e4db2aa8241a9\cmd;C:\Users\pmichalina\AppData\Local\GitHub\PortableGit_284a859b0e6deba86edc624fef1e4db2aa8241a9\usr\bin;C:\Users\pmichalina\AppData\Local\GitHub\PortableGit_284a859b0e6deba86edc624fef1e4db2aa8241a9\usr\share\git-tfs;C:\Users\pmichalina\AppData\Local\Apps\2.0\3MO5QPB8.1JA\K5RPAZTJ.3E9\gith..tion_317444273a93ac29_0003.0003_92b520eb113e6614;C:\Users\pmichalina\AppData\Local\GitHub\lfs-amd64_1.3.1;C:\Users\pmichalina\AppData\Local\Apps\2.0\3MO5QPB8.1JA\K5RPAZTJ.3E9\gith..tion_317444273a93ac29_0003.0003_92b520eb113e6614\NativeBinaries\x86;C:\Program Files\Haskell\bin;C:\Program Files\Haskell Platform\7.10.3\lib\extralibs\bin;C:\Program Files\Haskell Platform\7.10.3\bin;C:\Python27\;C:\Python27\Scripts;C:\ProgramData\Oracle\Java\javapath;C:\windows\system32;C:\windows;C:\windows\System32\Wbem;C:\windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\Client SDK\ODBC\130\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\Tools\Binn\ManagementStudio\;C:\Program Files\Git\cmd;C:\Program Files (x86)\Windows Kits\8.1\Windows Performance Toolkit\;C:\Program Files\Haskell Platform\7.10.3\mingw\bin;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\110\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files\Microsoft SQL Server\120\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\120\Tools\Binn\ManagementStudio\;C:\Program Files (x86)\Microsoft SQL Server\120\DTS\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\ProgramData\chocolatey\bin;C:\Program Files\dotnet\;C:\Program Files (x86)\WebEx\Productivity Tools;C:\Program Files (x86)\Microsoft SQL Server\110\DTS\Binn\;C:\Program Files\nodejs\;C:\Program Files (x86)\Sennheiser\SoftphoneSDK\;C:\Users\pmichalina\AppData\Roaming\cabal\bin;C:\Ruby23-x64\bin;C:\Program Files (x86)\Microsoft VS Code\bin;C:\Users\pmichalina\AppData\Roaming\npm;C:\Program Files (x86)\MSBuild\14.0\bin\;C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1A\bin\NETFX 4.5.1 Tools\x64
10 verbose lifecycle @SecretOrg/kick-ass-project@0.2.8~ttt: CWD: C:\Users\pmichalina\Source\Repos\pjmichalina\kick-ass-project
11 silly lifecycle @SecretOrg/kick-ass-project@0.2.8~ttt: Args: [ '/d /s /c', 'semantic-release "pre"' ]
12 silly lifecycle @SecretOrg/kick-ass-project@0.2.8~ttt: Returned: code: 1  signal: null
13 info lifecycle @SecretOrg/kick-ass-project@0.2.8~ttt: Failed to exec ttt script
14 verbose stack Error: @SecretOrg/kick-ass-project@0.2.8 ttt: `semantic-release "pre"`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (C:\Users\pmichalina\AppData\Roaming\npm\node_modules\npm\lib\utils\lifecycle.js:255:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (C:\Users\pmichalina\AppData\Roaming\npm\node_modules\npm\lib\utils\spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:852:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
15 verbose pkgid @SecretOrg/kick-ass-project@0.2.8
16 verbose cwd C:\Users\pmichalina\Source\Repos\pjmichalina\kick-ass-project
17 error Windows_NT 10.0.10586
18 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\pmichalina\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "ttt" "--" "pre"
19 error node v6.3.1
20 error npm  v3.10.8
21 error code ELIFECYCLE
22 error @SecretOrg/kick-ass-project@0.2.8 ttt: `semantic-release "pre"`
22 error Exit status 1
23 error Failed at the @SecretOrg/kick-ass-project@0.2.8 ttt script 'semantic-release "pre"'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the @SecretOrg/kick-ass-project package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     semantic-release "pre"
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs @SecretOrg/kick-ass-project
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls @SecretOrg/kick-ass-project
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]

```

**Comment by stevemao** [comment]
Are you on Windows? Is it related to https://github.com/conventional-changelog/standard-version/issues/101?

**Comment by patrickmichalina** [comment]
@stevemao Yes, it appears to be the same issues as #101. I am running windows.

**Comment by patrickmichalina** [comment]
I see similiar messages as the ones posted in #101 

```
warning: LF will be replaced by CRLF in CHANGELOG.md.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in package.json.
The file will have its original line endings in your working directory.
```

Is there a quick fix to this?

**Comment by Tapppi** [comment]
Closed as duplicate of #101
